designer/server/src/test/scala/pl/touk/nussknacker/ui/process/deployment/scenariostatus/InconsistentStateDetectorTest.scala

```diff
@@ -0,0 +1,98 @@
+package pl.touk.nussknacker.ui.process.deployment.scenariostatus
+
+import org.scalatest.funsuite.AnyFunSuiteLike
+import org.scalatest.matchers.should.Matchers
+import pl.touk.nussknacker.engine.api.deployment.{DeploymentStatusDetails, ScenarioActionName}
+import pl.touk.nussknacker.engine.api.deployment.simple.SimpleStateStatus
+import pl.touk.nussknacker.engine.api.deployment.simple.SimpleStateStatus.ProblemStateStatus
+import pl.touk.nussknacker.engine.deployment.DeploymentId
+
+import java.util.UUID
+
+// TODO: more unit tests, tests on higher level than (resolveScenarioStatus) not extractAtMostOneStatus
+class InconsistentStateDetectorTest extends AnyFunSuiteLike with Matchers {
```

Comment:
These tests were done in `FlinkDeploymentManagerSpec`. I moved them here, because there is no access to `InconsistentStateDetector` in that place.